pcweb/components/docpage/sidebar/sidebar.py

```diff
@@ -136,7 +136,7 @@ def sidebar_icon(name):
         "Wrapping React": "atom",
         "Vars": "variable",
         "Events": "arrow-left-right",
-        "Substates": "boxes",
+        "Component State": "boxes",
```

Comment:
This section should be called State Structure 

- Accessing State
- Component States